spec/services/stats/r003_business_unit_performance_report_spec.rb

```diff
@@ -430,24 +429,24 @@ module Stats
               %q{Performance %,Total received,Responded - in time,Responded - late,Open - in time,Open - late,} +
               %q{Performance %,Total received,Responded - in time,Responded - late,Open - in time,Open - late}
             expected_text = <<~EOCSV
-              Business unit report (FOIs) - 1 Jan 2017 to 30 Jun 2017
+              Business unit report (FOIs) - 5 Jun 2017 to 30 Jun 2017
```

Comment:
Why did we change this date from Jan to June? Does it just make the test faster or something? :)

Reply:
@willmcb , doesn't make it faster, but it will affect the outcome (the scope of cases), previous one doesn't test the scope properly which means the tests don't cover the important part of testing report, ( we need to count right scope of the cases based on the time frame)